### ISSUE TITLE
Use an image for success

### ISSUE THREAD

**Issue by joejag** [build-canaries/nevergreen#46]
If builds are all green. Allow the user to specify a URL for an image to load and displayed fullscreen.

**Comment by GentlemanHal** [comment]
Which takes precedence, the success text or url?

**Comment by joejag** [comment]
Url, and we can visually disable the success text input box when a URL is
entered?

**Comment by GentlemanHal** [comment]
I think ultimately this should be combined with #9 and we should just auto detect if it's a url or just plain text. We could make that change under issue #9, so to finish this issue I agree the url should just take precedence.

Although it might be easier to just explain that with some text on the page, given that tab is so sparse right now and we are potentially (if you agree) going to change it when we play #9.

**Comment by joejag** [comment]
There's an affinity between the two issues, but we can release this without completing #9 with too much rework right?

I like the idea of adding the textual description on the success page. That's something we need in general, and this one is a good place to start with.

Let's add text for completing this issue saying the URL takes precedence. Then on issue #9 let's write the text for what will appear on the screen before we start coding.

**Comment by cowley05** [comment]
agree with all!

**Comment by cowley05** [comment]
Ok so this is done to the best of my ability :s please QA this and see if its ok for next release...

**Comment by GentlemanHal** [comment]
Should we check the image is actually an image and can be loaded? The image I was testing with has disappeared and now I just get the broken image icon which looks crap.

**Comment by joejag** [comment]
We've got a preview that shows while they are configuring it.

A broken image logo when unavailable isn't too bad, I suppose we could say
that in text instead, but a broken image send that message.

What are our thoughts for doing something better?

**Comment by GentlemanHal** [comment]
Can't really think of anything other than manually making a call to the url and checking the status code and content type.

I agree for this issue we shouldn't worry but it might be worth looking into for the future as showing a full screen broken image for success would make Nevergreen look bad (even though the user has selected a broken link)

We could try doing something nicer in issue #9, as that would allow us to just skip showing broken links.

**Comment by cowley05** [comment]
Think this is pretty much there but needs @mrgeorgegray to sort out the scalling issue if you use funny sized images. I think as it is though we could release it like this and put another issue in next release or so to fix it?

**Comment by GentlemanHal** [comment]
I agree this issue can be closed and we can open another if we decide the scaling issue is worth fixing.

**Comment by cowley05** [comment]
think i may have fixed it :)
